### ISSUE TITLE
Is the copyright and license info on the footer still accurate? 

### ISSUE THREAD

**Issue by mastermatt** [expressjs/expressjs.com#1166]
The footer currently lists this info (in English, but most of the data is consistent across locales):
- Documentation translations provided by StrongLoop/IBM.
- Express is a project of the OpenJS Foundation.
- Edit this page on GitHub.
- Copyright © 2017 StrongLoop, IBM, and other expressjs.com contributors.
- Creative Commons License This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 United States License.

I'm fuzzy on the details of the StrongLoop/IBM/OpenJS F./Linux F. transition. Does StrongLoop still hold the copyrights to some/all of Express? Does this cover the lib or just the website? 

And I see that this repo and the site hasn't updated its license since 2016. I don't have reason to question this expect that it was before the transition and Express itself is MIT. Can someone give me the layman's tl;dr on having different licenses?

**Comment by jonchurch** [comment]
Most of these are before my time, but the only one that stands out to me here is the Copyright notice of StrongLoop/IBM. I'm not sure how that works exactly, but I think the notice is probably still accurate in regards to the website content.

For the Creative Commons license, there's some discussion here https://github.com/expressjs/expressjs.com/issues/634 and here https://github.com/expressjs/expressjs.com/issues/413

The TLDR is that CC license is intended for content (works of art, text, etc) vs MIT which is for software specifically. It covers the website, not the library.

Express is an OpenJS project, so I know that's accurate.

**Comment by crandmck** [comment]
Since I was there, maybe I can shed some light.  Caveat: I'm not a lawyer or legal expert by any means....

I agree, as @jonchurch said, I think the copyright applies to the website content... It's there because IBM/StrongLoop paid salaries for several people (including myself) to completely overhaul the documentation over the course of a couple months.  We started with what TJ wrote, which had some good technical info, but was all in one giant page, not well organized or written, nor easy to navigate, and was somewhat outdated.  We converted it into the site you see today, and added quite a bit of material (e.g. "Getting Started", lots of "Guides", etc).  IBM also paid to translate the content (professional technical translation is very expensive).  Since then, there has been _extensive_ improvement and maintenance by @dougwilson and many others in the community; but IIUC that doesn't change the original copyright.

Although IBM handed over the domain name to LF some time ago, I believe the copyright notice should remain as is; but again, I'm not a lawyer.  I guess we could remove "StrongLoop" since that company ceased to exist as an entity several years ago. However, I have a personal fondness for keeping it, since I was part of the scrappy little startup which was subsequently swallowed up by IBM.  But if anyone feels strongly it should be removed, I can't think of a really valid objection.

BTW, I don't mean to take any undue credit... While I did do a lot of work in the initial site creation/conversion, many others contributed as well. As with any OS project it was a team effort.

Upshot: I think the footer is fine "as is."

**Comment by gireeshpunathil** [comment]
> I guess we could remove "StrongLoop" since that company ceased to exist as an entity several years ago. However, I have a personal fondness for keeping it, since I was part of the scrappy little startup which was subsequently swallowed up by IBM.

+1 to keeping it (Strongloop part in IBM/Strongloop) as is. My rational: It shows the genesis (through second token), while technically correct (through the first one)
